tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -524,8 +526,27 @@ static void stream_qos_set(struct bt_bap_stream *stream)
 static void stream_enabled(struct bt_bap_stream *stream)
 {
 	struct audio_stream *a_stream = CONTAINER_OF(stream, struct audio_stream, stream);
+	struct bt_bap_ep_info info;
+	struct bt_conn_info conn_info;
+	int err;
 
 	LOG_DBG("Enabled stream %p", stream);
+
+	err = bt_bap_ep_get_info(stream->ep, &info);
+	if (err) {
+		LOG_ERR("Could not get EP info for stream %p", stream);
+		return;
+	}
+
+	(void)bt_conn_get_info(stream->conn, &conn_info);
```

Comment:
Any reason why we are ignoring the return value of `bt_conn_get_info` but not of `bt_bap_ep_get_info`? 

IMO makes sense to treat these 2 the same